src/flow/src/adapter/flownode_impl.rs

```diff
@@ -92,6 +95,35 @@ impl Flownode for FlowWorkerManager {
                     .map_err(to_meta_err)?;
                 Ok(Default::default())
             }
+            Some(flow_request::Body::Flush(FlushFlow {
+                flow_id: Some(flow_id),
+            })) => {
+                // TODO(discord9): impl individual flush
+                debug!("Starting to flush flow_id={:?}", flow_id);
```

Comment:
Can't flush for a single flow currently?

Reply:
> Can't flush for a single flow currently?

Sadly no, run flow will compute all flow with available inputs anyway, and since all results will be buffered in sink buffer, don't want to make sink buffer too large